src/java/Main.java

```diff
@@ -0,0 +1,49 @@
+package java;
+
+public class Main {
+
+    public static void main(String[] args) {
+        Civil AN=new Civil();
+        AN.BusinessClass=false;
+        AN.NumberOfPassengers=20;
+        AN.maksSpeed=700;
+        AN.MinLength=250;
+        AN.power=1000;
+        AN.WingSpan=50;
+        AN.Vyvod();
+        AN.Power();
+        AN.loading();
+        Military MIG= new Military();
```

Comment:
@Hlieb имена объектов/экземпляров класса всегда с маленькой буквы
```suggestion
        Military mIG= new Military();
```